src/AbstractNavItem.tsx

```diff
@@ -93,10 +93,13 @@ const AbstractNavItem: AbstractNavItem = React.forwardRef(
     }
 
     if (props.role === 'tab') {
-      props.tabIndex = isActive ? props.tabIndex : -1;
       props['aria-selected'] = isActive;
     }
 
+    if (props.disabled) {
+      props.tabIndex = -1;
+    }
```

Comment:
i'm not sure this is right. I think we only want to insist on this in the tab case. OItherwise it's necessarily true that items styled as disabled should not be tabbable, especially if a tabindex was provided by the user explicitly

Reply:
and we should probably add aria-disabled in the case where it is disabled

Reply:
> i'm not sure this is right. I think we only want to insist on this in the tab case. OItherwise it's necessarily true that items styled as disabled should not be tabbable, especially if a tabindex was provided by the user explicitly

Changed to allow user overrides for tabIndex in ListGroupItem but not in tabs.  Also added aria-disabled.